app/views/secrets/show.html.erb

```diff
@@ -119,6 +119,17 @@ $(function () {
         $warn.text('Value is not valid json ( ' + e.toString() + ' )');
         $warn.show();
       }
+    } else if(/^\s+/.test(val) || /\s+$/.test(val)) {
```

Comment:
how about putting that under `app/assets/javascripts/whitespace_stripper.js` and then reuse it for env vars ... we have the same problem there :)